src/tp245.jl

```diff
@@ -0,0 +1,24 @@
+# TP problem 245 in NLS format
+#
+#   Source:
+#   Klaus Schittkowski,
+#   More test examples for nonlinear programming codes,
+#   Lecture Notes in Economics and Mathematical Systems 282,
+#   Springer Verlag Berlin Heidelberg, 1987
+#   10.1007/978-3-642-61582-5
+#
+# A. Montoison, Montréal, 06/2018.
+
+export tp245
+
+"Test problem 245 in NLS format"
+function tp245()
+
+  nls = Model()
+  x0  = [0; 10; 20]
+  @variable(nls, x[i=1:3], start=x0[i])
+
+  @NLexpression(nls, F[i=1:10], exp(-i*x[1]/10) - exp(-i*x[2]/10) - x[3]*(exp(-i/10) - exp(-i)))
```

Comment:
tp242 and tp245 have the same `F`, but there it uses `t` and here it doesn't. I think it's better if we use `t` (or not) uniformly.